src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft Azure PowerBI Dedicated Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for PowerBI Dedicated Resources.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
@selasagi make it compatible with nuget version to 0.9.0 for Assembly and fileversion

Reply:
The link to the spec - https://github.com/Azure/azure-rest-api-specs/tree/current/specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/2017-10-01/powerbidedicated.json
it is merged. Thanks